src/net_processing.cpp

```diff
@@ -3164,7 +3167,14 @@ bool SendMessages(CNode* pto, CConnman& connman, const std::atomic<bool>& interr
                 return true;
             }
         }
+        MsgGetData(consensusParams, pto, connman, msgMaker, state, nNow, fFetch);
+        MsgFeeFilter(pto, connman, msgMaker);
+    }
+    return true;
+}
 
+void MsgGetData(const Consensus::Params& consensusParams, CNode* pto, CConnman& connman, const CNetMsgMaker& msgMaker, CNodeState& state, int64_t nNow, bool fFetch)
```

Comment:
Maybe make `nNow` and `fFetch` const, to ensure they are not mistakenly updated (and discarded on return) in this method.